crates/oxc_formatter/src/utils/conditional.rs

```diff
@@ -344,9 +344,9 @@ impl<'a> FormatConditionalLike<'a, '_> {
 
     /// Checks if any part of the conditional has multiline comments
     #[inline]
-    fn has_multiline_comment(_f: &Formatter<'_, 'a>) -> bool {
-        // TODO: Implement multiline comment detection
-        false
+    fn has_multiline_comment(&self, f: &Formatter<'_, 'a>) -> bool {
+        let span = self.span();
+        f.comments().has_comment_in_range(span.start, span.end)
```

Comment:
It doesn't check any comments to see whether they have multiple lines. 

I don't understand the logic in [Biome](https://github.com/biomejs/biome/blob/ceed34a164cc76406baf617000b8877160f0bcc3/crates/biome_js_formatter/src/utils/conditional.rs#L197-L219), and no tests fail by this, so I didn't implement it. Can you double what it does? Is there any case that would be affected by this?

Reply:
Take a look at Prettier as well, maybe it is already dead code

Reply:
Remove them then! File an issue about this in Prettier if you'd like.

Reply:
OK, I'll:

- Revert some useless change here to focus fixing bug
- Remove `has_multiline_comment` and related lines
- Report to prettier